### ISSUE TITLE
autobean.share.link broken in 0.2.0

### ISSUE THREAD

**Issue by laermannjan** [SEIAROTg/autobean#8]
First of all, thanks for your awesome plugins: I've been loving the `share` feature a lot so far!

I've installed `autobean==0.2.0` recently and now the `autobean.share.link` directive will always complain that the specified ledgers have not been imported with `autobean.share.include` despite them being imported right before.
This was working fine in `0.1.3`.

```
; household.bean
plugin "beancount.plugins.auto_accounts"
2000-01-01 *
    Assets:External:Alice        -50.00 USD ; we don't care exactly what bank
        share-Alice: 1
    Expenses:Movie                40.00 USD
        share-Alice: 1
        share-Bob: 1
    Expenses:Popcron              10.00 USD
        share-Alice: 1

; alice.bean
plugin "beancount.plugins.auto_accounts"
2000-01-01 *
    Assets:BoA:Checking          -50.00 USD
      share-Alice: 1
    Assets:External:Joint
      share-Joint: 1

; alice-view.bean
option "operating_currency" "USD"
plugin "autobean.share" "Alice"
plugin "beancount.plugins.auto_accounts"
2000-01-01 custom "autobean.share.include" "household.bean"
2000-01-01 custom "autobean.share.include" "alice.bean"
2000-01-02 custom "autobean.share.link" "household.bean" Assets:External:Alice "alice.bean" Assets:External:Joint
  ; error in fava:
  ; Ledger main.beancount was not included with "autobean.share.include" from this ledger
  ; Ledger jan.beancount was not included with "autobean.share.include" from this ledger
```

**Comment by SEIAROTg** [comment]
Thanks for the report!

It's an inconsistency in path normalization so it wasn't able to find the identical path. Now fixed in 740862d08885fef697f936b0a6cf22f8389670e6 and released in 0.2.1:

```
python -m beancount.reports.report alice-view.bean print
2000-01-01 open Assets:External:Alice
2000-01-01 open Assets:BoA:Checking
2000-01-01 open Expenses:Movie
2000-01-01 open Assets:External:Joint
2000-01-01 open Expenses:Popcron
2000-01-01 open Assets:Receivables:Bob

2000-01-01 *
  Expenses:Movie           20.00 USD
  Expenses:Popcron         10.00 USD
  Assets:BoA:Checking     -50.00 USD
  Assets:Receivables:Bob   20.00 USD
```

**Comment by laermannjan** [comment]
Just wanted to leave a big thank you for the super quick response and immediate fix!
